chirpstack/src/storage/search.rs

```diff
@@ -161,6 +161,17 @@ pub async fn global_search(
     Ok(res)
 }
 
+#[cfg(feature = "sqlite")]
+pub async fn global_search(
+    _user_id: &Uuid,
+    _global_admin: bool,
+    _search: &str,
+    _limit: usize,
+    _offset: usize,
+) -> Result<Vec<SearchResult>, Error> {
+    unimplemented!()
```

Comment:
@momobel I think this function still needs to be implemented. Is that something you are still working on?

Reply:
It is now implemented on Sqlite.